backend/ticketvise/views/api/__init__.py

```diff
@@ -1 +1,24 @@
+from rest_framework import serializers
+
 AUTOCOMPLETE_MAX_ENTRIES = 10
+
+
+class DynamicFieldsModelSerializer(serializers.ModelSerializer):
+    """
+    A ModelSerializer that takes an additional `fields` argument that
+    controls which fields should be displayed.
+        """
+
+    def __init__(self, *args, **kwargs):
+        # Don't pass the 'fields' arg up to the superclass
+        fields = kwargs.pop('fields', None)
+
+        # Instantiate the superclass normally
+        super(DynamicFieldsModelSerializer, self).__init__(*args, **kwargs)
+
+        if fields:
+            # Drop any fields that are not specified in the `fields` argument.
+            allowed = set(fields)
+            existing = set(self.fields.keys())
+            for field_name in existing - allowed:
+                self.fields.pop(field_name)
```

Comment:
Nice deze class, kunnen we die fields niet in de frontend meegeven? Ook misschien deze class verplaatsen naar de __ init __.py van de models of de utils.py class

Reply:
In het frontend meegeven is gevaarlijk, omdat je dan requests kan doen waar je geen recht op hebt.

Reply:
Ja dat snap ik. Ik bedoelde dat we binnen die velden kunnen aangeven wat we daarvan willen gebruiken, om minder data te gebruiken.